cmake/ycm-0.4.0/modules/AddInstallRPATHSupport.cmake

```diff
@@ -100,6 +99,19 @@ function(ADD_INSTALL_RPATH_SUPPORT)
                              "${_multiValueArgs}"
                              "${ARGN}")
 
+  # if either RPATH is disabled or INSTALL_RPATH is disable
```

Comment:
Should be *disable**d** at the end of the comment, or rephrased as:
"if either RPATH or INSTALL_RPATH is disabled"